### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.12 es 6.2 (25 novembre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#331] [Esercizi M-Z]
<img width="944" alt="image" src="https://github.com/user-attachments/assets/3ead853b-452c-4e2f-882c-de202f16a7e7">

**Comment by AglaiaNorza** [comment]
![es](https://github.com/user-attachments/assets/b7c17d52-7546-4e4b-ac8f-4a3f3774c26d)
